src/yunohost/app.py

```diff
@@ -1901,10 +1901,10 @@ def _fetch_app_from_git(app):
                 # we will be able to use it. Without this option all the history
                 # of the submodules repo is downloaded.
                 subprocess.check_call([
-                    'git', 'clone', '--depth=1', '--recursive', url,
+                    'git', 'clone', '--recursive', url,
```

Comment:
Can you elaborate on the removal of `--depth=1` here ? I don't understand how that's related to the issue :/

Reply:
or more simple : git clone -b <branch> <remote_repo>

Reply:
An error of me, sorry.